blivet/formats/fs.py

```diff
@@ -445,27 +448,28 @@ def do_check(self):
 
         self._fsck.do_task()
 
-    def load_module(self):
-        """Load whatever kernel module is required to support this filesystem."""
-        if not self._modules or self.mount_type in kernel_filesystems:
+    def check_module(self):
+        """Check if kernel module required to support this filesystem is available."""
+        if not self._modules or self.mount_type in AVAILABLE_FILESYSTEMS:
             return
 
         for module in self._modules:
+            # try modprobe --dry-run to check if the module is available
             try:
-                rc = util.run_program(["modprobe", module])
+                rc = util.run_program(["modprobe --dry-run", module])
```

Comment:
This could use `blockdev.utils_have_kernel_module()`.

Reply:
Good point, I've changed the code to use libblockdev.